### PR TITLE
Cleanup Typography + utilize new Typography system from Unicorn UI

### PR DESCRIPTION
**Pull request by Noxsios**
## Description

Utilizes new Typography system from @mike-winberry : https://unicorn-ui.netlify.app/theme

- Replaces current global theming via `app.css` w/ an implementation in `typography.ts`
- Performs overrides to stay in harmony with the Figma spec laid out by @Madeline-UX  

### Additions:

- Remove dead `Container` and `Modal` components
- Replace any leftover `<p>` and `<hX` elements w/ Typography

### CODE REVIEW

**Comment by jeff-mccoy** [comment]
Is some of this text supposed to be white?
Screenshot 2023-01-12 at 10 47 40 PM
Screenshot 2023-01-12 at 10 47 50 PM

Is the "Package Not Found" supposed to be so huge?
Screenshot 2023-01-12 at 10 47 55 PM

File: src/ui/routes/initialize/+layout.svelte [hunk 1]
```diff
@@ -55,7 +55,7 @@
 	{:else if status == LoadingStatus.Error}
 		<!-- replace w/ error dialog -->
 		<div class="center">
-			<Typography variant="h1">Package Not Found</Typography>
+			<Typography variant="h4">Package Not Found</Typography>
```
Comment (jeff-mccoy) [inline src/ui/routes/initialize/+layout.svelte:58]: 😆  that is all.

**Review by jeff-mccoy — COMMENTED** [review]
changes all look reasonable, can we get a few screenshots of the changes here before approval?

**Comment by Noxsios** [comment]
https://user-images.githubusercontent.com/50058333/213015506-522a8288-61bf-484d-8016-c9318ffd6c5f.mov

@jeff-mccoy

**Review by jeff-mccoy — APPROVED** [review]
@Noxsios raising the bar on async PR review process. Also lgtm.

**Pull request merged**